best-practices/native/nativescript/evironment.md

```diff
@@ -0,0 +1,80 @@
+The best practice recommends not use more than 2 properties, production and apiUrl properties.
```

Comment:
Couple of suggested changes here:

```suggestion
The best practice recommends not to use more than 2 properties, `production` and `apiUrl`.
```

I had to look it up and its called "splitting infinitives" to use `not to`.